app/views/documents/form_partials/_upload.html.erb

```diff
@@ -1,14 +1,12 @@
 <% if @document.new_record? %>
   <%= form.govuk_file_field(
         :file,
-        label: {text: t(".upload_a_file")},
-        direct_upload: true
+        label: {text: t(".upload_a_file")}
```

Comment:
We must have used direct uploads in the past but Rails code protected us from ourselves